### ISSUE TITLE
a11y: color palette does not meet AA guidelines

### ISSUE THREAD

**Issue by clshortfuse** [angular/material#10164]
**Actual Behavior**:
 -  `What is the issue? *` Current color palette uses old Angular Material opacities for colors that do not meet a11y guidelines.
 -  `What is the expected behavior?` Should use new values.

**Angular Versions**: *
 -  `Angular Material Version:` 1.1.1

----

We are currently using old opacity values which used 87% opacity for certain colors:

https://web.archive.org/web/20140628054353/http://www.google.com/design/spec/style/color.html#color-ui-color-application

We should be using 100% opacity as per the new guidelines. https://material.google.com/style/color.html#color-text-background-colors

For example, default text for `grey-600` (#757575) should now be 100% white, not 87% white. This moves the contrast ratio from `3.9` to `4.6`. 

https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html states we should require at least `4.5` contrast ratio for all primary text.

**Comment by Splaktar** [comment]
The spec updates for 2017 can still be found here:
https://material.io/archive/guidelines/style/color.html#color-usability

This archive holds the details prior to the MD 2018 changes.

**Comment by Splaktar** [comment]
Just a note here to make sure that we check both light and dark contrasts as mentioned in https://github.com/angular/material/issues/8992.
